sample-app.yaml

```diff
@@ -0,0 +1,50 @@
+# event-display app deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: python-ceph-vectordb
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: python-ceph-vectordb
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      containers:
+        - name: python-ceph-vectordb
```

Comment:
my guuess is that we would need a container per bucket (all using the same image)?
so, we probably need to name something like `python-ceph-vectordb-bucket-text`?